### ISSUE TITLE
Refactor project structure

### ISSUE THREAD

**Issue by adamzerella** [govau/chameleon#9]
Ideally we should have:

```
- src
- src/test ( server side unit tests )
- templates
- test ( Dev testing with dummy template )
```

**Comment by alex-page** [comment]
I would always bundle tests together. As a developer it is not intuitive to have tests hidden in the `src` folder.

**Comment by adamzerella** [comment]
How should we seperate the static `index.html` testing and server unit tests ?

**Comment by alex-page** [comment]
Good question! I think something like this:

```
/test/
/test/unit/
/test/integration/
/test/system/
```

Unit - Contains functional tests, using something like jest
Integration - Actually hits the server and compares what is sent back
System - A basic html page with a form

**Comment by adamzerella** [comment]
Resolved after mering #30
